internal/telemetry/properties/properties.go

```diff
@@ -69,3 +82,20 @@ func (g PropertyGetter) ClusterProperties() (p ClusterProperties) {
 	}
 	return
 }
+
+func (g PropertyGetter) RepositoryProperties() (p RepositoryProperties) {
+	if g.RepositoryLister != nil {
+		if ls, err := g.RepositoryLister.ListRepositories(context.Background()); err == nil {
+			p.nrepositories = len(ls.Items)
+			for _, repo := range ls.Items {
+				if repo.Spec.Auth != nil {
+					p.nrepositoriesAuth = p.nrepositoriesAuth + 1
+				}
+				if repo.IsDefaultRepository() && !strings.Contains(repo.Spec.Url, "https://packages.dl.glasskube.dev/packages") {
```

Comment:
Just two minor things:
1. Maybe we should extract the default URL to a constant? Put it in `/internal/constants/`?
2. Is there a reason you use `string.Contains` vs a simple `!=`?

Reply:
ad 2.: I thought just to be safe about the trailing slash. Someone could tinker around and change the url or delete the repo, then change it back or recreate it and do it without the trailing slash. Granted, a very very specific case.

Reply:
Fair enough, we can keep it that way